src/components/atoms/InputLabel/InputLabel.tsx

```diff
@@ -1,21 +1,53 @@
-import React, { ReactNode, FC } from 'react';
+import React, { HTMLAttributes, FC } from 'react';
 import styled from 'styled-components';
-import Text from '../Text/Text';
-import { colors } from '../../../constants/colors';
+import { typography } from '../../../constants/typography';
+import { colors, INeutralColorSpec, ISemanticColorSpec } from '../../../constants/colors';
 
-export interface IInputLabelProps {
+export interface IInputLabelProps extends HTMLAttributes<HTMLLabelElement> {
+  /**
+   * The weight of the rendered text. Avoid the use of `semibold` as for now.
+   */
+  weight?: keyof typeof typography.weights;
+  /**
+   * The size you want to render your text at, currently only `14px` | `16px` and `20px` supported.
+   */
+  size?: keyof typeof typography.sizes.text | 'lead';
+  /**
+   * Whether to add some margin below the rendered text or not. Use it to give meaningful white-space.
+   */
+  mb?: boolean;
+  /**
+   * Accepts a subset of the Zopa brand colors. Same as the ones accepted by `<Heading />`.
+   */
+  color?:
+    | INeutralColorSpec['white']
+    | INeutralColorSpec['nearDark']
+    | INeutralColorSpec['dark']
+    | ISemanticColorSpec['success']
+    | ISemanticColorSpec['error'];
```

Comment:
@moshie lets just leave `<InputLabe />` with the `htmlFor` prop ( _CSS looks good_ 👍  )

Although previously you could do something like `<InputLabe size="small" />`, it didn't have any effect as `<Text />` props were locked through the `attrs` syntax: `styled(Text).attrs({})`

Reply:
@lluia Just to double check are we still keeping the margin bottom and color?

Like this?:

```jsx
export interface IInputLabelProps extends HTMLAttributes<HTMLLabelElement> {
  /**
   * Define the associated input identifier
   */
  htmlFor: string;
}

const InputLabel = styled.label<IInputLabelProps>`
  display: block;
  margin: 0;
  letter-spacing: 0;
  color: ${colors.neutral.dark};
  margin-bottom: 24px;
  line-height: ${typography.lineHeights.text};
  font-family: ${typography.primary};
  font-weight: ${typography.weights.semibold};
  font-size: 'base';
`;
```